src/uiTest/resources/project.rhino.js

```diff
@@ -237,4 +245,15 @@ function Project(underlyingProject) {
     if (hash === null) return null;
     return hash.asString();
   };
+
+  this.isInSyncToParentStatus = function (parent, child, status) {
```

Comment:
a check for `InSync` would be enough ofc however I decided to provide a bit more general method so we can check any statuses in the future

Reply:
:benissimo:

Reply:
Maybe even just rewrite to `getSyncToParentStatus` which would return a string, to be compared against the desired value in Scala test code?

Reply:
Even if the current impl stays, note that it should rather be called
```suggestion
  this.hasGivenSyncToParentStatus = function (parent, child, status) {
```

`InSync` in the name suggest a check specifically for a green edge